cmd/channel/email/main.go

```diff
@@ -5,31 +5,43 @@ import (
 	"database/sql"
 	"encoding/json"
 	"fmt"
+	"github.com/emersion/go-sasl"
+	"github.com/emersion/go-smtp"
 	"github.com/icinga/icinga-notifications/internal"
 	"github.com/icinga/icinga-notifications/pkg/plugin"
 	"github.com/icinga/icingadb/pkg/types"
+	"github.com/jhillyerd/enmime"
 	"net"
-	"net/smtp"
+	"net/mail"
 	"os"
 	"os/user"
-	"strings"
+)
+
+const (
+	TypeNone     = "none"
+	TypeStartTls = "starttls"
+	TypeTls      = "tls"
```

Comment:
Why `Type...`? I'd name them something like `EncryptionNone`. Also, acronyms are typically treated special (see for example [`http.ParseHTTPVersion`](https://pkg.go.dev/net/http#ParseHTTPVersion)), so it should be `EncryptionStartTLS` and `EncryptionTLS`.